### PR TITLE
Added fonts and left-sidebar

### PR DESCRIPTION
**Pull request by Setup1337**
image
Example

### CODE REVIEW

File: frontend/src/shared/components/left-sidebar/left-sidebar.component.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            </svg>
+            <span>Settings</span>
+          </a>
+        </div>
+      </li>
+    </ul>
+
```
Comment (ismv) [inline frontend/src/shared/components/left-sidebar/left-sidebar.component.html:94]: remove empty space pls

File: frontend/src/shared/components/left-sidebar/left-sidebar.component.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+
```
Comment (ismv) [inline frontend/src/shared/components/left-sidebar/left-sidebar.component.html]: remove empty space pls

File: frontend/src/shared/components/left-sidebar/left-sidebar.component.sass [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  /* white/100
+
+  background: #FFFFFF
+
+  /* Menu
+
+  box-shadow: 3px 4px 9px rgba(223, 223, 223, 0.25)
+
+.logo
+  background: #F0F3F9
```
Comment (ismv) [inline frontend/src/shared/components/left-sidebar/left-sidebar.component.sass]: use our colorscheme here

File: frontend/src/shared/components/left-sidebar/left-sidebar.component.sass [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  height: 100%
+  position: fixed
+  z-index: 1
+  overflow-x: hidden
+  left: 0
+  top: 0
+
+  /* white/100
+
+  background: #FFFFFF
```
Comment (ismv) [inline frontend/src/shared/components/left-sidebar/left-sidebar.component.sass]: use our colorscheme here

File: frontend/src/shared/components/left-sidebar/left-sidebar.component.sass [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  box-shadow: 3px 4px 9px rgba(223, 223, 223, 0.25)
+
+.logo
+  background: colorscheme(gray-200)
+  width: 87.63px
+  height: 57.23px
+  border-radius: 10px
+  margin-left: 51px
+  margin-top: 32.34px
+
```
Comment (FeloniukM) [inline frontend/src/shared/components/left-sidebar/left-sidebar.component.sass]: Why is there white space here?

File: frontend/src/shared/components/left-sidebar/left-sidebar.component.sass [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  vertical-align: middle
+
+.icon
+  margin-left: 50px
+  margin-top: 76.45px
+
+.menu
+  margin-left: 49px
+  margin-top: 48.01px
+  margin-right: 48px
```
Comment (FeloniukM) [inline frontend/src/shared/components/left-sidebar/left-sidebar.component.sass:35]: and here there is none?

Comment (Setup1337) [reply frontend/src/shared/components/left-sidebar/left-sidebar.component.sass:35]: Added whitespace

Comment (andpii) [reply frontend/src/shared/components/left-sidebar/left-sidebar.component.sass:35]: It's good to have this setting enabled in vs code to not bother anymore with white spaces
image

**Review by ismv — CHANGES_REQUESTED** [review]
Conflicts resolving

File: frontend/src/shared/components/left-sidebar/left-sidebar.component.html (hunk 2 above)
Comment (Setup1337) [reply frontend/src/shared/components/left-sidebar/left-sidebar.component.html]: Done

File: frontend/src/shared/components/left-sidebar/left-sidebar.component.sass (hunk 3 above)
Comment (Setup1337) [reply frontend/src/shared/components/left-sidebar/left-sidebar.component.sass]: Done

File: frontend/src/styles.sass [hunk 7]
```diff
@@ -1,3 +1,29 @@
...
+  font-family: 'Nunito', sans-serif
+  font-style: normal
+  font-weight: 700
+  font-size: 18px
+h6
+  font-family: 'Nunito', sans-serif
+  font-style: normal
+  font-weight: 700
+  font-size: 16px
+
```
Comment (ismv) [inline frontend/src/styles.sass]: remove empty space

Comment (Setup1337) [reply frontend/src/styles.sass]: Done

**Review by ismv — APPROVED** [review]

**Review by andpii — APPROVED** [review]

**Pull request merged**